client/android/app/src/main/AndroidManifest.xml

```diff
@@ -12,10 +12,6 @@
         <action android:name="android.intent.action.MAIN" />
         <category android:name="android.intent.category.LAUNCHER" />
       </intent-filter>
-      <intent-filter>
-        <action android:name="FLUTTER_NOTIFICATION_CLICK" />
-        <category android:name="android.intent.category.DEFAULT" />
-      </intent-filter>
```

Comment:
Don't we need this to send notifications?

Reply:
Search for "FLUTTER_NOTIFICATION_CLICK" on the migration docs. That's where they suggest to remove it: https://firebase.flutter.dev/docs/migration/. I still need to go through a process of verifying that notifications work. I haven't done that.